CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Added`
 
 - [#290](https://github.com/nf-core/funcscan/pull/290) Merged pipeline template of nf-core/tools version 2.9, updated references. (by @jfy133)
+- [#285](https://github.com/nf-core/funcscan/pull/285) Added support for `fna.gz` input FASTA files (by @louperelo, @mirpedrol, @jfy133)
```

Comment:
Is this description correct?

Reply:
Maybe "create input channel with Channel.fromSamplesheet"? Or similar. 
But this one sounds good too, I didn't see you changed the pattern and was confused :)

Reply:
```suggestion
- [#285](https://github.com/nf-core/funcscan/pull/285) Use nf-validation for samplesheet checking and added support for `fna.gz` input FASTA files (by @louperelo, @mirpedrol, @jfy133)
```